src/visualization/types/Mosaic/view.tsx

```diff
@@ -64,14 +59,13 @@ const MosaicPlot: FunctionComponent<Props> = ({
 
   const [yDomain, onSetYDomain, onResetYDomain] = useVisYDomainSettings(
     properties.yDomain,
-    result.table.getColumn(yColumn, 'string')
+    result.table.getColumn(ySeriesColumns[0], 'string')
   )
 
   const isValidView =
     xColumn &&
     columnKeys.includes(xColumn) &&
-    yColumn &&
-    columnKeys.includes(yColumn) &&
+    ySeriesColumns.every(col => columnKeys.includes(col)) &&
```

Comment:
```suggestion
    ySeriesColumns.every(yCol => columnKeys.includes(yCol)) &&
```

just provides a little extra context when we're dealing with y column and x column

Reply:
Good point. I changed it to the singluar. It's more verbose as `ySeriesColumn` but also makes clear that Mosaic uses a special kind of yColumn and not the yColumn used in other visualizations.